doc/cprover-manual/contracts-frees.md

```diff
@@ -0,0 +1,120 @@
+[CPROVER Manual TOC](../../)
+
+# Frees Clauses
+
+A _frees clause_ allows the user to specify a set of pointers that may be freed
+by a function. A function contract may have zero or more frees clauses.
```

Comment:
Is that just this function that has the contract attached to it, or any function that may (transitively) be called by the function that has the contract?

Reply:
transitively